docs/ops/activation/Mish.md

```diff
@@ -4,9 +4,9 @@
 
 **Category**: *Activation*
 
-**Short description**: Mish a Self Regularized Non-Monotonic Neural Activation Function.
+**Short description**: Mish is a Self Regularized Non-Monotonic Neural Activation Function.
 
-**Detailed description**: Mish layer computes self regularized non-monotonic neural activation function: x * tanh(softplus(x))
+**Detailed description**: Mish layer computes self regularized non-monotonic neural activation function: x * tanh(softplus(x)) [Reference](https://arxiv.org/abs/1908.08681)
```

Comment:
Please, remove the formula in this line and change the text to:
`...non-monotonic neural activation function proposed in the [article](https://arxiv.org/abs/1908.08681).`